chia/rpc/data_layer_rpc_api.py

```diff
@@ -159,3 +161,39 @@ async def get_roots(self, request: Dict[str, Any]) -> Dict[str, Any]:
             raise Exception("Data layer not created")
         res = await self.service.get_roots(store_ids)
         return {"hashes": res}
+
+    async def subscribe(self, request: Dict[str, Any]) -> bool:
+        """
+        rows_to_add a list of clvm objects as bytes to add to talbe
+        rows_to_remove a list of row hashes to remove
+        """
+        store_id = request.get("id")
+        if store_id is None:
+            raise Exception("missing store id in request")
+        ip = request.get("ip")
+        if ip is None:
+            raise Exception("missing ip in request")
+        port = request.get("port")
+        if port is None:
+            raise Exception("missing port in request")
+        latest = request.get("latest")
```

Comment:
add mode as a string in the dict? have {"id", "mode", "ip", "port"} as a request, and default "mode" to "history"?

Reply:
this is what im doing, but the flag is called latest

Reply:
I'm suggesting adding a string instead of a flag, called "mode". This will help if, at some point, we decide to support mode modes other than "latest" and "history".